src/Util/UrlUtil/UrlUtil.spec.js

```diff
@@ -73,6 +73,22 @@ describe('UrlUtil', () => {
       });
     });
 
+    describe('#getQueryParam', () => {
+      it('is defined', () => {
+        expect(UrlUtil.getQueryParam).toBeDefined();
+      });
+      it('returns the value of the given query param of the provided URL', () => {
+        let got = UrlUtil.getQueryParam('http://borussia.de/bvb', 'bvb');
+        expect(got).toEqual(undefined);
+
+        got = UrlUtil.getQueryParam('http://borussia.de/bvb?bvb=09', 'bvb');
+        expect(got).toEqual('09');
+
+        got = UrlUtil.getQueryParam('http://borussia.de/bvb?bvb=09&shinji=kagawa', 'shinji');
+        expect(got).toEqual('kagawa');
+      });
```

Comment:
What do we get back from

```
UrlUtil.getQueryParam('http://example.com/foo?humpty=1&humpty=2', 'humpty')
```
?

Reply:
Actually it returns `2` as the query string gets parsed by the underlying modules `url-parse`/`querystringify` to `{ humpty: '2' }`.

Reply:
OK. I *think* a return value of `[1, 2]` would be more correct. See e.g. this library: https://github.com/sindresorhus/query-string.

Reply:
Yes, I agree. Switched from `querystringify` to `query-string` to get this done. See ca71de7.